test/new-e2e/pkg/runner/parameters/const.go

```diff
@@ -50,3 +50,9 @@ const (
 	// DevMode config flag parameter name
 	DevMode StoreKey = "dev_mode"
 )
+
+// AWSoverrides is a map of StoreKey to StoreKey used to override key only in AWS store
+var AWSoverrides = map[StoreKey]StoreKey{
+	APIKey: "api_key_2",
+	APPKey: "app_key_2",
+}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Would rather have per store overrides, so having these two only for `store_aws`, to avoid impacting local usage

Reply:
I see these are indeed only used by `store_aws.go`, I would move this there, even if at the end of the day this is the same package